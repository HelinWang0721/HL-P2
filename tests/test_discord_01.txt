a, b;
main {

    // a = b;
        // public: a ;



    public: b;
    private: a;
    b = a;
    a = x;
}
