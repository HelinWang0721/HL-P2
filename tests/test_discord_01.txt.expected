main.b = main.a
main.a = ?.x
